seqr/views/apis/summary_data_api.py

```diff
@@ -87,8 +87,11 @@ def saved_variants_page(request, tag):
     if tag == 'ALL':
         saved_variant_models = SavedVariant.objects.exclude(varianttag=None)
     else:
-        tag_type = VariantTagType.objects.get(name=tag, project__isnull=True)
-        saved_variant_models = SavedVariant.objects.filter(varianttag__variant_tag_type=tag_type)
+        tags = tag.split(';')
+        tag_type = VariantTagType.objects.filter(name__in=tags, project__isnull=True)
```

Comment:
should be `tag_types` plural, as it is a collection of models not a single model